php/src/Controller/DockerController.php

```diff
@@ -185,7 +185,7 @@ public function StartContainer(Request $request, Response $response, array $args
 
         $config = $this->configurationManager->GetConfig();
         // set AIO_URL
-        $config['AIO_URL'] = $host . ':' . $port . $path;
+        $config['AIO_URL'] = $host . ':' . (string)$port . $path;
```

Comment:
```suggestion
        $config['AIO_URL'] = $host . ':' . strval($port) . $path;
```
Personally, I prefer `strval` to casting; I think casting is ugly.